pkg/blockstorage/helpers.go

```diff
@@ -105,3 +106,143 @@ func FilterSnapshotsWithTags(snapshots []*Snapshot, tags map[string]string) []*S
 	}
 	return result
 }
+
+// utility functions equivalent to old functions from package `go-autorest/autorest/to`
+
+// StringMapPtr returns a pointer to a map of string pointers built from the passed map of strings.
+func StringMapPtr(ms map[string]string) *map[string]*string {
+	msp := make(map[string]*string, len(ms))
+	for k, s := range ms {
+		msp[k] = azto.Ptr(s)
+	}
+	return &msp
+}
+
+// StringMap returns a map of strings built from the map of string pointers. The empty string is
+// used for nil pointers.
+func StringMap(msp map[string]*string) map[string]string {
+	ms := make(map[string]string, len(msp))
+	for k, sp := range msp {
+		if sp != nil {
+			ms[k] = *sp
+		} else {
+			ms[k] = ""
+		}
+	}
+	return ms
+}
+
+// StringSlice returns a string slice value for the passed string slice pointer. It returns a nil
+// slice if the pointer is nil.
+func StringSlice(s *[]string) []string {
+	if s != nil {
+		return *s
+	}
+	return nil
+}
+
+// StringSlicePtr returns a pointer to the passed string slice.
+func StringSlicePtr(s []string) *[]string {
+	return azto.Ptr(s)
+}
+
+// SliceStringPtr returns a slice of string pointers from the passed string slice.
+func SliceStringPtr(s []string) []*string {
+	ms := make([]*string, len(s))
+	for k, sp := range s {
+		ms[k] = azto.Ptr(sp)
+	}
+	return ms
+}
+
+// Bool returns a bool value for the passed bool pointer. It returns false if the pointer is nil.
+func Bool(b *bool) bool {
+	if b != nil {
+		return *b
+	}
+	return false
+}
+
+// BoolPtr returns a pointer to the passed bool.
+func BoolPtr(b bool) *bool {
+	return &b
```

Comment:
Why not use `azto.Ptr`?

For example:
```suggestion
	return azto.Ptr(b)
```

And, why couldn't `azto.Ptr` be directly called at the caller? instead of the `*Ptr` functions defined here?

Reply:
@julio-lopez , you are correct about using azto.ptr 
Now to avoid spamming, removed